kpatch-build/kpatch-elf.h

```diff
@@ -76,6 +76,7 @@ struct symbol {
 	struct section *sec;
 	GElf_Sym sym;
 	char *name;
+	struct object_symbol *lookup_table_sym;
```

Comment:
Technically this should probably be called `lookup_table_file_sym`, since it points to the lookup table's FILE symbol rather than the corresponding symbol itself.

That said, maybe it would be simpler to just set it to the exact corresponding symbol to begin with.  locals_match() already finds the corresponding symbol -- twice, in fact.  Then, later, lookup_local_symbol() finds it a third time.  Can we just set `sym->lookup_table_sym` to the corresponding symbol to begin with, in find_local_syms() and/or locals_match()?  Then lookup_local_symbol() becomes redundant.

Reply:
Ah, I see. That makes sense, I'll see if it will work.

Reply:
After looking into this I currently don't see a straightforward way of dropping `lookup_local_symbol()`. The only reason is `sympos` calculation it does. Because of that we'll either have to traverse the whole lookup table for each symbol in `find_local_syms()` (which will make things worse) or keep a list of seen symbol names and write down sympos in `symtab_read()` but I am not sure it is worth doing that. Any thoughts? Can we leave it as is for now?